documentation/flows/agent/ap_metrics.md

```diff
@@ -5,22 +5,23 @@ The AP metrics can optionally include station metrics and statistics, as configu
 
 # Messages
 
-Access Point Metrics should be represented as 2 messages:
+Access Point Metrics should be represented as 3 messages:
 
-* AP Metrics query
-* AP Metrics response
+* AP Metrics Query
+* AP Metrics Response
+* Multi-AP Policy Config Request
 
-## AP Metrics query
+## AP Metrics Query
 
-`AP Metrics query` message has type `0x800b` and tlv type `0x93`.
+`AP Metrics Query` message has type `0x800b` and tlv type `0x93`.
 
 TLV payload are:
 
 * List of BSSIDs for which the metrics are to be reported.
 
-## AP Metrics response
+## AP Metrics Response
 
-`AP Metrics response` message has type `0x800c`.
+`AP Metrics Response` message has type `0x800c`.
 It has one or more AP Metric TLVs with type `0x94` (one per BSSID reported).
```

Comment:
While you're correcting things: it's actually called AP Metric***s*** TLV. I made that mistake :-)

Reply:
Fixed